### PR TITLE
Improved delta t correction for even pixels

### PR DESCRIPTION
**Pull request by pawel21**
Hi, 

I've improved code to ```time_lapse_corr``` for even pixel/channel in each DRS4.

### CODE REVIEW

File: lstchain/calib/camera/r0.py [hunk 1]
```diff
@@ -346,6 +346,22 @@ def do_time_lapse_corr(waveform, expected_pixel_id, local_clock_list, fc, last_t
...
+                # now the magic of Dragon,
+                # if the ROI is in the last quarter of each DRS4
+                # for even channel numbers extra 12 slices are read in a different place
+                # code from Takayuki & Julian
+                if pix % 2 == 0:
+                    first_cap = fc[nr_module, gain, pix]
+                    if first_cap % 1024 > 766 and first_cap % 1024 < 1012:
+                        start = int(first_cap) + 1024 - 1
+                        end = int(first_cap) + 1024 + 11
+                        for kk in range(start, end):
```
Comment (vuillaut) [inline lstchain/calib/camera/r0.py]: I think you may avoid the loop here with
`last_time_array[nr_module, gain, pix, start%4096:end%4096] = time_now`

**Comment by rlopezcoto** [comment]
Hi @pawel21, as soon as you implement @vuillaut comments we can go ahead and merge before you pull the code that you just presented today at the call.

File: lstchain/calib/camera/r0.py (hunk 1 above)
Comment (pawel21) [reply lstchain/calib/camera/r0.py]: Done, thank you very much.

**Comment by rlopezcoto** [comment]
Hi @pawel21, it seems that there are some conflicts with some of the extractors that have changed name in ctapipe... sorry, but could you please have a look and correct that? Thanks

**Comment by vuillaut** [comment]
> Hi @pawel21, it seems that there are some conflicts with some of the extractors that have changed name in ctapipe... sorry, but could you please have a look and correct that? Thanks

This is fixed by PR #122. When merged, you can merge master again.

**Comment by rlopezcoto** [comment]
Thanks @vuillaut. #122 is merged already, but now there is the "Roque de los Muchachos" error popping up

**Comment by vuillaut** [comment]
Hey @rlopezcoto 
Just restarted the build. Tests are passing.

**Comment by rlopezcoto** [comment]
Damn, I did it twice and got the same error... anyways, with #123 it should be solved

**Pull request merged**